### PR TITLE
[#246] Style the Upload & Submit button, header logo and textbox for …

### PR DESCRIPTION
**Pull request by yasminegha**
…moment creation page.

### CODE REVIEW

File: frontend/src/styles/CreateMoment.css [hunk 1]
```diff
@@ -1,31 +1,227 @@
...
+  font-size: 48px;
+}
 .content-container {
+    display: block;
+    position: center;
+    margin-top: 30px;
     margin-bottom: 20px;
-}
 
+}
```
Comment (ZacharyBys) [inline frontend/src/styles/CreateMoment.css]: whitespace

File: frontend/src/styles/CreateMoment.css [hunk 2]
```diff
@@ -1,31 +1,227 @@
...
     margin-right: 40px;
 }
 
-.title { 
-    width: 300px;
-}
-
 .textarea {
     padding-left: 10px;
+
```
Comment (ZacharyBys) [inline frontend/src/styles/CreateMoment.css]: whitespace

File: frontend/src/pages/MomentCreationPage.js [hunk 3]
```diff
@@ -16,6 +15,7 @@ class MomentCreation extends Component {
         }
     }
 
+
```
Comment (ZacharyBys) [inline frontend/src/pages/MomentCreationPage.js]: whitespace

File: frontend/src/styles/CreateMoment.css [hunk 4]
```diff
@@ -1,31 +1,227 @@
...
+    border-radius: 2px;
+}
+.button:hover{
+    border-width: 2px;
+    border-color: #1DE28F;
+}
+/*
+.upload-btn-wrapper input[type=file] {
+  font-size: 100px;
+  position: absolute;
```
Comment (ZacharyBys) [inline frontend/src/styles/CreateMoment.css]: what is this?

File: frontend/src/styles/CreateMoment.css [hunk 5]
```diff
@@ -1,31 +1,227 @@
...
+}
+*/
+.inputfile {
+  cursor: pointer;
+	width: 0.1px;
+	height: 0.01px;
+	overflow: hidden;
+	position: absolute;
+	z-index: -1;
+  background-color: white;
```
Comment (ZacharyBys) [inline frontend/src/styles/CreateMoment.css]: sometimes you write #FFFFFF and sometimes white. choose one and make it consistent

File: frontend/src/styles/CreateMoment.css [hunk 6]
```diff
@@ -1,31 +1,227 @@
...
+  --inputTransitionTF: var(--transitionTF);
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
```
Comment (ZacharyBys) [inline frontend/src/styles/CreateMoment.css]: Why define a fontsize for every element on the page when you're defining individual font sizes?

File: frontend/src/styles/CreateMoment.css [hunk 7]
```diff
@@ -1,31 +1,227 @@
...
+  font-size: 24px;
+  font-weight: 300;
+  line-height: 1.1;
+  border: none;
+  border-radius: 6.5px;
+  transition: box-shadow 0.3s;
+}
+
+.Input-text:focus {
+
```
Comment (ZacharyBys) [inline frontend/src/styles/CreateMoment.css]: whitespace

File: frontend/src/styles/CreateMoment.css [hunk 8]
```diff
@@ -1,31 +1,227 @@
...
+  opacity: 0;
+  transform:
+    translate3d(0, 50%, 0)
+    scale(1);
+  transform-origin: 0 0;
+
+}
+
+.Input-text:placeholder-shown + .Input-label {
+
```
Comment (ZacharyBys) [inline frontend/src/styles/CreateMoment.css]: whitespace

File: frontend/src/styles/CreateMoment.css [hunk 9]
```diff
@@ -1,31 +1,227 @@
...
+  visibility: hidden;
+  z-index: -1;
+}
+
+.Input-text:not(:placeholder-shown) + .Input-label,
+.Input-text:focus:not(:placeholder-shown) + .Input-label {
+
```
Comment (ZacharyBys) [inline frontend/src/styles/CreateMoment.css]: whitespace

File: frontend/src/styles/CreateMoment.css [hunk 10]
```diff
@@ -1,31 +1,227 @@
...
+  font-family: "Teko";
+  font-size: 28px;
+  font-weight: 300;
+  line-height: 1.8;
+  opacity: 0;
+  transform:
+    translate3d(0, 50%, 0)
+    scale(1);
+  transform-origin: 0 0;
+
```
Comment (ZacharyBys) [inline frontend/src/styles/CreateMoment.css]: whitespace

**Review by ZacharyBys — DISMISSED** [review]

File: frontend/src/pages/MomentCreationPage.js [hunk 11]
```diff
@@ -41,23 +41,46 @@ class MomentCreation extends Component {
...
+    }
+
     render(){
         return(
             <div>
                 <NavBar user={this.state.user}/>
+                <div className = "createMoment-header">
+                <img
+                    alt="UploadMoment"
+                    src={`http://${process.env.REACT_APP_IP}/res/uploadMoment.png`}
```
Comment (tehblasian) [inline frontend/src/pages/MomentCreationPage.js]: The image is pixelated... can't have that

File: frontend/src/pages/MomentCreationPage.js [hunk 12]
```diff
@@ -41,23 +41,46 @@ class MomentCreation extends Component {
...
-                        <div className = "content-container">
-                            <input type="file" name="file" required></input>
+                            <input type="file" name="file" id="file" className="inputfile" onChange={this.handleChange} required/>
+                            <label for="file">Choose Moment
+                                <img
+                                    alt="inputFile"
+                                    src={`http://${process.env.REACT_APP_IP}/res/inputfile.png`}
+                                />
+                            </label>
+                            <div id="filename"></div>
```
Comment (tehblasian) [inline frontend/src/pages/MomentCreationPage.js]: Remove this empty div

File: frontend/src/styles/PageHeader.css [hunk 13]
```diff
@@ -0,0 +1,54 @@
+* {
+    margin: 0;
+    padding: 0;
+  }
+
+  html, body{
+    height: 100%;
```
Comment (ZacharyBys) [inline frontend/src/styles/PageHeader.css]: Take off the general tags like this

File: frontend/src/pages/MomentCreationPage.js [hunk 14]
```diff
@@ -41,23 +41,46 @@ class MomentCreation extends Component {
...
                 <NavBar user={this.state.user}/>
+                <div className = "createMoment-header">
+                <img
+                    alt="UploadMoment"
+                    src={`http://${process.env.REACT_APP_IP}/res/uploadMoment.png`}
+                />
+                </div>
                 <div className = "CreateMoment">
                     {this.state.redirect && <Redirect to={this.state.redirect} />}
                     <form onSubmit={this.handleSubmit} encType="multipart/form-data" noValidate>
```
Comment (tehblasian) [inline frontend/src/pages/MomentCreationPage.js]: In the form, you don't need to wrap every single element/group of elements with a div. remove the content-containers

File: frontend/src/pages/MomentCreationPage.js [hunk 15]
```diff
@@ -41,23 +41,46 @@ class MomentCreation extends Component {
...
+                            <div id="filename"></div>
                         </div>
                         <div className = "content-container">
-                            <Header textSize={2} text="Description" />
-                            <textarea name="description" rows="8" cols="90"></textarea>
+                            <div className = "createMoment-description">
+                              Description
+                            </div>
+                            <div className="Wrapper">
+                                <div className="Input">
```
Comment (tehblasian) [inline frontend/src/pages/MomentCreationPage.js]: You dont need to wrap the textarea and label with a div

File: frontend/src/pages/MomentCreationPage.js [hunk 16]
```diff
@@ -41,23 +41,46 @@ class MomentCreation extends Component {
...
+                                    <textarea type="text" id="input" name="description" className="Input-text" row="4" cols="50" placeholder="Add a Description to your Moment"/>
+                                    <label for="input" className="Input-label">This will help users come up with your ideal caption!</label>
+                                </div>
+                            </div>    
                         </div>
                         <div className = "content-container">
```
Comment (tehblasian) [inline frontend/src/pages/MomentCreationPage.js]: Remove the wrapping div

File: frontend/src/pages/MomentCreationPage.js [hunk 17]
```diff
@@ -41,23 +41,46 @@ class MomentCreation extends Component {
...
+                                    alt="inputFile"
+                                    src={`http://${process.env.REACT_APP_IP}/res/inputfile.png`}
+                                />
+                            </label>
+                            <div id="filename"></div>
                         </div>
                         <div className = "content-container">
-                            <Header textSize={2} text="Description" />
-                            <textarea name="description" rows="8" cols="90"></textarea>
+                            <div className = "createMoment-description">
```
Comment (tehblasian) [inline frontend/src/pages/MomentCreationPage.js]: This should be a `<label>`. Also, it should be placed right above the input field on the left side.
see https://uxdesign.cc/design-better-forms-96fadca0f49c

**Review by tehblasian — DISMISSED** [review]

File: frontend/src/pages/MomentCreationPage.js (hunk 11 above)
Comment (yasminegha) [reply frontend/src/pages/MomentCreationPage.js]: fixed it

File: frontend/src/pages/MomentCreationPage.js (hunk 12 above)
Comment (yasminegha) [reply frontend/src/pages/MomentCreationPage.js]: done

**Review by ZacharyBys — APPROVED** [review]
Decent.

**Pull request merged**